dark-node/dark_node.go

```diff
@@ -30,7 +30,8 @@ import (
 )
 
 // Prime is the default prime number used to define the finite field.
-var Prime, _ = big.NewInt(0).SetString("179769313486231590772930519078902473361797697894230657273430081157732675805500963132708477322407536021120113879871393357658789768814416622492847430639474124377767893424865485276302219601246094119453082952085005768838150682342462881473913110540827237163350510684586298239947245938479716304835356329624224137859", 10)
+var primeVal, _ = stackint.FromString("179769313486231590772930519078902473361797697894230657273430081157732675805500963132708477322407536021120113879871393357658789768814416622492847430639474124377767893424865485276302219601246094119453082952085005768838150682342462881473913110540827237163350510684586298239947245938479716304835356329624224137111")
```

Comment:
Previously, the prime being used was the first prime greater than 2^1024. This is too big to be stored in a stackint, so the prime has been changed to the first prime _smaller_ than 2^1024.